tpkg/verify_timeout.tdir/verify_timeout.test

```diff
@@ -0,0 +1,22 @@
+# #-- verify_timeout.test --#
+# source the master var file when it's there
+[ -f ../.tpkg.var.master ] && source ../.tpkg.var.master
+# use .tpkg.var.test for in test variable passing
+[ -f .tpkg.var.test ] && source .tpkg.var.test
+. ../common.sh
+
+# start NSD
+get_random_port 1
+TPKG_PORT=$RND_PORT
+
+PRE="../.."
+NSD="$PRE/nsd"
+
+${NSD} -c nsd.conf -L 5 -F 0xFFFF -p ${NSD_PORT}
```

Comment:
Test if the `-L` option is available.  Like in other tests that need the debug functionality.